docs/specifying-a-call-to-configure.md

```diff
@@ -110,6 +110,23 @@ A.CallTo(fakeShop).Where(call => call.Method.Name == "set_Address")
                   .Throws(new Exception("we can't move"));
 ```
 
+## Specifying a call to an event accessor
+
+Although calls to event accessors can be specified using the approach described
+in the previous section, FakeItEasy also provides helper methods to make this
+easier:

```

Comment:
nit: the first two examples below _are_ the approach described above

Reply:
What do you mean? The examples below use the new `EventAction` feature

Reply:
No, I am clearly having some sort of brain deficiency. I read the "set_" and "get_" above and I guess translated to "add_" and "remove_". After _carefully_ rereading, I see what you have here is perfect. 😊